go/api/generic_commands.go

```diff
@@ -30,4 +30,275 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Exists returns the number of keys that exist in the database
+	//
+	// Note:
+	//When in cluster mode, the command may route to multiple nodes when keys map to different hash slots.
```

Comment:
There is new format of notice was introduced in #2587. Please update it here and for all commands already implemented (which support multi-slot routing).

Reply:
done